core-tests/src/test/java/org/mule/runtime/core/internal/el/dataweave/DataWeaveExpressionLanguageAdaptorTestCase.java

```diff
@@ -649,7 +649,8 @@ public void compilationExceptionPropagates() {
   @Test
   public void evaluateInvalidCompiledExpression() throws MuleException {
     ExpressionCompilationException e = new ExpressionCompilationException(createStaticMessage("oopsy"));
-    expectedEx.expect(is(sameInstance(e)));
+    expectedEx.expect(ExpressionRuntimeException.class);
```

Comment:
we should assert thet the error message is meaningful here

Reply:
no point in doing that. I'm asserting that the thrown assertion is the same as in the mock